test-data/unit/cmdline-based-baseline.test

```diff
@@ -98,6 +98,19 @@ No errors, baseline file removed
 Success: no issues found in 1 source file
 == Return code: 0
 
+[case testRewriteNoChange]
+# cmd: mypy --write-baseline --error-summary pkg
+[file pkg/a.py]
+a
+[file .mypy/baseline.json]
+{
+"__baseline_metadata__": {"targets": ["file:pkg"]},
+"pkg/a.py": [{"code": "name-defined", "column": 0, "line": 1, "message": "Name \"a\" is not defined", "target": "a"}]
+}
+[out]
+Found 1 error (0 new errors) in 1 file (checked 1 source file)
+Baseline successfully written to .mypy/baseline.json
+== Return code: 0
```

Comment:
Could you make this case separated with double newlines

Reply:
or is it that some test cases are meant to have no newline?

Reply:
Thanks, I only meant that test, but all is a bonus.